device-types/Cisco/WS-C4506.yaml

```diff
@@ -9,14 +9,24 @@ subdevice_role: parent
 console-ports:
   - name: con0
     type: rj-45
-power-ports:
-  - name: PS1
-    type: iec-60320-c14
-    maximum_draw: 1500
-  - name: PS2
-    type: iec-60320-c14
-    maximum_draw: 1500
 interfaces:
   - name: FastEthernet1
     type: 100base-tx
     mgmt_only: true
+module-bays:
```

Comment:
This looks like a parent device, is that so? If so, these also appear that it might have devices that slot into this chassis. If that is indeed true, then you should have all applicable device slots under `device-bays` rather than `module-bays` which are reserved for things like PSUs, Fans, SFP interface cards, etc. I don't use these so I do not know, but it looks like this should have device bays

Reply:
4506 is a chassis switch.  The parent device role needs to be removed.